src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -226,14 +226,22 @@ await requestExecutor.ExecuteAsync(_redirectNode, null, context, command, should
 
                 var databaseName = _store.GetDatabase(_dbName);
 
+                bool compressionSupport = false;
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Where this is defined? Probably best place would be Directory.Build.props
Also it is better to do it the other way around since the number of unsupported target frameworks is more or less constant and number of supported ones will grow.

Reply:
Done